manifests/components/kibana.jsonnet

```diff
@@ -38,18 +39,69 @@ local strip_trailing_slash(s) = (
     },
   },
 
+  // List of Kibana plug-ins to install
+  plugins:: {
+    /* Example:
+    logtrail: {
+      version: "v0.1.31",
+      url: "https://github.com/sivasamyk/logtrail/releases/download/v0.1.31/logtrail-6.7.1-0.1.31.zip",
+    },
+    */
+  },
+
   es: error "elasticsearch is required",
 
   serviceAccount: kube.ServiceAccount($.p + "kibana") + $.metadata {
   },
 
+  pvc: kube.PersistentVolumeClaim($.p + "kibana-plugins") + $.metadata {
+    storage: "2Gi",
+  },
+
   deploy: kube.Deployment($.p + "kibana") + $.metadata {
     spec+: {
       template+: {
         spec+: {
           securityContext: {
             fsGroup: 1001,
           },
+          volumes_+: {
+            plugins: kube.PersistentVolumeClaimVolume($.pvc),
```

Comment:
This is a Deployment using a (ReadWriteOnce) PVC.  This will be ok, provided we only have a single replica (and kube.libsonnet already adjusts the update strategy accordingly for this situation).
Do we ever want multiple kibana replicas?

Reply:
To be honest, I don't see a good reason to have multiple Kibana replicas. But since the PVC is only used to keep the plug-ins, we could use a 1-1 mapping between a Kibana Pod and the PVC used for plug-ins, right?

Reply:
sg, happy to leave it as-is based on the idea that we think this will only ever be replicas=1.  If we ever decide to change that, then we can revisit this at that time.